controlplane/kubeadm/internal/etcd/etcd.go

```diff
@@ -68,7 +68,7 @@ type MemberAlarm struct {
 type AlarmType int32
 
 const (
-	// AlarmOK denotes that the cluster member is OK.
+	// AlarmOk denotes that the cluster member is OK.
 	AlarmOk AlarmType = iota
```

Comment:
shouldn't we actually rename the variable to `AlarmOK` according to convention and [speeling](https://en.wikipedia.org/wiki/OK#:~:text=OK%20(spelling%20variations%20include%20okay,or%20a%20sign%20of%20indifference.)?

Reply:
I think you're right

Reply:
I will fix.